core/jwks/README.md

```
@@ -0,0 +1,91 @@
+npx vitest run tests/basic.test.ts
+
+# Integration tests (with live Clerk endpoint)
+npx vitest run tests/integration.test.ts
+
+# All tests
+./test-all.sh
+```
+
+## Structure
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)